doc/library/tensor/nnet/ctc.txt

```diff
@@ -0,0 +1,27 @@
+.. _libdoc_tensor_nnet_ctc:
+
+==============================================================
+:mod:`ctc` -- Connectionist Temporal Classification (CTC) loss
```

Comment:
As @abergeron suggested, and as you do in `doc/library/gpuarray/ctc.txt`, use the full module name here (`theano.tensor.nnet.ctc`).